### DISCUSSION TITLE
Auto Mesh integration?

### DISCUSSION THREAD

**Discussion by NoobToNirvana** [dchristl/esp32_nat_router_extended#123] [Ideas]
Hi,

The 'Auto Mesh' feature is inbuilt in the code of martin-ger but his code is not compatible with esp32-s2-mini . Your work has the esp-s2-mini chip compatibility and they can be powerful mesh devices. Any chance you might consider auto mesh integration in the future?

I tested your existing code in esp32-s2-mini and it works wonderfully. Excellent job and many thanks.

**Comment by dchristl** [comment]
Hello,

I would love to continue working on the project, and the mesh feature sounds sensible. Unfortunately, I absolutely lack the time for it. However, I will mark the feature and add it when I have more time.

Kind Regards,
Danny

**Reply by NoobToNirvana** [reply]
Hi Danny,

Thank you for making it in your to-do list. Excellent work and thank you for taking your time to keep improving this excellent project. 

I am not qualified to contribute to this area of coding. Many thanks and have a good week ahead :)

Best Regards,
NN
